include/psa/crypto_values.h

```diff
@@ -1129,6 +1129,23 @@
  */
 #define PSA_ALG_OFB                             ((psa_algorithm_t)0x04c01200)
 
+/** The CCM* cipher mode without authentication.
+ *
+ * This is CCM* as specified in IEEE 802.15.4 §7, with a tag length of 0.
+ * For CCM* with a nonzero tag length, use the AEAD algorithm #PSA_ALG_CCM.
+ *
+ * The underlying block cipher is determined by the key type.
+ *
+ * CCM* requires knowledge of the plaintext length before producing output.
```

Comment:
CCM-star in unauthenticated mode does not require prior knowledge of the plaintext length, only about the size of the IV. It basically reduces to AES-CTR with a CTR-IV of `L' || CCM-IV || (L' + 1) * 0x00` (where L' is the one-byte encoding of L, which in turn is the amount of bytes in the full-block IV reserved for length encoding, where L is allowed to be between 2 and 7 inclusive).

So by setting the IV length one has successfully constrained CCM_STAR_NO_TAG. That precludes the use of the single-part interface, though, unless the value of L becomes part of the algorithm encoding?